wave_lang/kernel/wave/constraints.py

```diff
@@ -234,6 +234,7 @@ class HardwareConstraint(Constraint):
     mma_type: Optional[MMAType | ScaledMMAType] = MMAType.F32_16x16x16_F16
     vector_shapes: Optional[dict[IndexSymbol, int]] = None
     max_bits_per_load: int = 128
+    use_linearized_cta_dims: Optional[bool] = None
```

Comment:
Can you rename this to use_linearized_dims? cta has not been defined elsewhere and so seems a bit out of place.